pkg/virtctl/create/vm/vm.go

```diff
@@ -1086,39 +1153,48 @@ func withVolumeSourceImageIO(paramStr string) (*cdiv1.DataVolumeSpec, *uint, err
 		return nil, nil, err
 	}
 
-	if src.URL == "" || src.DiskId == "" {
+	if src.URL == "" || src.DiskID == "" {
 		return nil, nil, params.FlagErr(VolumeImportFlag, "url and diskid are both required with imageio volume source")
 	}
 
 	return &cdiv1.DataVolumeSpec{
 		Source: &cdiv1.DataVolumeSource{
 			Imageio: &cdiv1.DataVolumeSourceImageIO{
 				URL:           src.URL,
-				DiskID:        src.DiskId,
+				DiskID:        src.DiskID,
 				SecretRef:     src.SecretRef,
 				CertConfigMap: src.CertConfigMap,
 			},
 		},
 	}, src.BootOrder, nil
 }
 
-func withVolumeSourcePVC(paramStr string) (*cdiv1.DataVolumeSpec, *uint, error) {
+func parseVolumeSource(paramStr, kind string) (name, namespace string, bootOrder *uint, err error) {
```

Comment:
Did the linter suggest to move it into a separate function?

Reply:
Yes, there were totally alike functions that differed only by return. So `dupl` linter was not happy with that.